docs/source/use_cases/compute_principal_coordinate_analysis/2-way-pca.rst

```diff
@@ -22,10 +22,7 @@ __ RenderedVersion_
 
 .. contents::
 
-`Principal Coordinate Analysis <http://occamstypewriter.org/boboh/2012/01/17/pca_and_pcoa_explained/>`_
-counts the number of variants two samples have in common.  These counts are then placed into an
-``NxN`` matrix where ``N`` is the number of samples in the variant set.  The matrix is centered,
-scaled, and then the first two principal components are computed for each invididual.
+`Principal Coordinate Analysis <http://occamstypewriter.org/boboh/2012/01/17/pca_and_pcoa_explained/>`_ counts the number of variants two samples have in common.  These counts are then placed into an ``(N+M)x(N+M)`` matrix where ``N`` is the number of samples in the control variant set (e.g., 1,000 Genomes) and ``M`` is the number of samples in the case variant set.  The matrix is centered, scaled, and then the first two principal components are computed for each invididual.
```

Comment:
Nicole, when performing this Multidimensional scaling I'm not sure you're actually doing any scaling in the code - as referenced in the link below - besides the centering before running the PCA on the centered dissimilarity matrix:

https://github.com/googlegenomics/spark-examples/blob/15213450109e0d8629317f98c275be1cf5114072/src/main/scala/com/google/cloud/genomics/spark/examples/VariantsPca.scala#L221

Hope it helps,
~p